catroid/src/main/java/org/catrobat/catroid/ui/dynamiclistview/UtilDynamicListView.java

```diff
@@ -608,7 +616,9 @@ public void notifyListItemTouchActionUp() {
 	private void touchEventsCancelled() {
 		if (cellIsMobile) {
 			View mobileView = getViewForID(mobileItemId);
-			unsetSpriteMobileState();
+			if (forSpriteList) {
+				unsetSpriteMobileState();
+			}
 			aboveItemId = INVALID_ID;
 			mobileItemId = INVALID_ID;
 			belowItemId = INVALID_ID;
```

Comment:
Could you please override the getAdapter method for the ListView with the following method:
@Override
	public ListAdapter getAdapter() {
		ListAdapter adapter = super.getAdapter();
		if (adapter instanceof HeaderViewListAdapter) {
			return ((HeaderViewListAdapter) adapter).getWrappedAdapter();
		}

		return adapter;
	}

This will fix the crash for DragNDrop in Scenes after copying (please verify)